SayTheirNames/Resources/he.lproj/Localizable.strings

```diff
@@ -0,0 +1,159 @@
+"say_their_names" = "אומר את השמות שלהם";
+
+/* loco:5ee29770f904c5151c1a206f */
+"bookmark" = "סימניות";
+
+/* loco:5ee29770f904c5151c1a2070 */
+"search" = "חיפוש";
+
+/* loco:5ee29770f904c5151c1a2071 */
+"donate" = "תרום";
+
+/* loco:5ee29770f904c5151c1a2072 */
+"DONATE NOW" = "תרום עכשיו";
+
+/* loco:5ee29770f904c5151c1a2073 */
+"description" = "הסבר";
+
+/* loco:5ee29770f904c5151c1a2074 */
+"person.say_their_names" = "אומר את השמות שלהם";
+
+/* loco:5ee29770f904c5151c1a2075 */
+"person.age" = "גיל";
+
+/* loco:5ee29770f904c5151c1a2076 */
+"person.children" = "ילדים";
+
+/* loco:5ee29770f904c5151c1a2077 */
+"person.location" = "אתר";
+
+/* loco:5ee29770f904c5151c1a2078 */
+"person.their_story" = "הסיפור שלהם";
+
+/* loco:5ee29770f904c5151c1a2079 */
+"person.outcome" = "תוצאה";
+
+/* loco:5ee29770f904c5151c1a207a */
+"person.news" = "חדשות";
+
+/* loco:5ee29770f904c5151c1a207b */
+"person.media" = "מדיה";
+
+/* loco:5ee29770f904c5151c1a207c */
+"person.hashtags" = "Social Media Hashtags";
+
+/* loco:5ee29770f904c5151c1a207d */
+"person.hashtagCopyTip" = "Press and hold to copy";
+
+/* loco:5ee29770f904c5151c1a207e */
+"person.copied" = "מועתק";
+
+/* loco:5ee29770f904c5151c1a207f
+ * Get Involved - Slack */
+"get_involved.slack.title" = "איך אני יכול לעזור?";
+
+/* loco:5ee29770f904c5151c1a2080 */
+"get_involved.slack.button" = "JOIN OUR SLACK CHANNEL";
+
+/* loco:5ee29770f904c5151c1a2082
+ * Get Involved - Developer */
+"get_involved.developer.title" = "HOW DO I GET INVOLVED AS A DEVELOPER?";
+
+/* loco:5ee29770f904c5151c1a2083 */
+"get_involved.developer.button" = "VIEW OUR GITHUB";
+
+/* loco:5ee29770f904c5151c1a2085
+ * Get Involved - Twitter */
+"get_involved.twitter.title" = "הצטרף אלינו בטוויטר";
+
+/* loco:5ee29770f904c5151c1a2086 */
+"get_involved.twitter.button" = "עקבו אחרינו";
+
+/* loco:5ee29770f904c5151c1a2088
+ * More */
+"more_history.aboutTitle" = "SOME HISTORY ON THE SAY THEIR NAMES MOVEMENT";
+
+/* loco:5ee29770f904c5151c1a2089 */
+"more_history.aboutDesc" = "Say Their Names is a social movement to raise awareness for all the black victims of racial inequality all around the world. This movement is in support of the overall Black Lives Matter movement and aims to highlight individuals affected by anti-Black Violence. Say Their Names movement calls attention and demands justice for all victims.";
+
+/* loco:5ee29770f904c5151c1a208a */
+"massive_thank_you" = "תודה ענקית לכל המעורבים";
+
+/* loco:5ee2ce86663b704aa41688f2 */
+"sign" = "חתום על עצומה זו";
+
+/* loco:5ee376d765812927aa78d0a2 */
+"about_this_project.title" = "על פרויקט זה";
+
+/* loco:5ee376dd6ca9905dc077cd32 */
+"about-this-project-details" = "Welcome to the Say Their Names Project. Our aim is to build an open-source platform to raise awareness of the injustice and often forgotten names of racial inequality. We strive to identify and amplify verified organizations to ensure donations are reaching those who can make the most impact with it.";
```

Comment:
Is this supposed not to be translated?

Reply:
On localise.biz it says Hebrew is 71% translated. Missing translations get substituted with English. @helloITdepartment were you planning on expanding the localizations, or leave it as is now?

Reply:
Ah sorry, will get the rest in asap

Reply:
Finished